### PR TITLE
Panda crossing

### PR DESCRIPTION
**Pull request by zumbak04**
## Developer changelog:
- Added river and sea crossings in Pandaria.

# How to test:
Play someone in Pandaria and move troops across rivers and sea straits.

### CODE REVIEW

**Comment by zumbak04** [comment]
> Sea province names and major river province names in the province view tooltip are wrong all over the map, not only in Pandaria. Other than that crossings seem to work fine.

Since Pandaria is superior, I fixed it in bd8d4e0e8f9ccd7b9ff3b849d06abf9109a76d6a.

File: map/adjacencies.csv [hunk 1]
```diff
@@ -181,3 +181,86 @@ From;To;Type;Through;-1;-1;-1;-1;Comment
...
+1190;1180;sea;1459;-1;-1;-1;-1;Vekam � Stinging Trail
+1190;1192;sea;1459;-1;-1;-1;-1;Vekam � Kypari Ik
+936;937;major_river;1424;-1;-1;-1;-1;Thundercleft � Krasarang
+934;865;major_river;1424;-1;-1;-1;-1;Narsong � Shen-zin
+604;557;major_river;1424;-1;-1;-1;-1;Fallingblossom � Cloudstrike
+1206;1202;sea;1475;-1;-1;-1;-1;Juman � Niuzao
+1205;1213;sea;1475;-1;-1;-1;-1;Vithurak � Sikvess
+1215;1202;sea;1475;-1;-1;-1;-1;Underbough � Niuzao
+1204;1208;sea;1478;-1;-1;-1;-1;Zhalmar � Kzzok
+1213;1186;major_river;1213;-1;-1;-1;-1;Sikvess � Rikkilea
```
Comment (Kyraga) [inline map/adjacencies.csv:223]: 1213;1186;major_river;1454;-1;-1;-1;-1;Sikvess � Rikkilea

File: map/adjacencies.csv [hunk 2]
```diff
@@ -181,3 +181,86 @@ From;To;Type;Through;-1;-1;-1;-1;Comment
...
+1213;1185;major_river;1213;-1;-1;-1;-1;Sikvess � Korvess
```
Comment (Kyraga) [inline map/adjacencies.csv:224]: 1213;1186;major_river;1454;-1;-1;-1;-1;Sikvess � Rikkilea
1213;1185;major_river;1454;-1;-1;-1;-1;Sikvess � Korvess

File: map/adjacencies.csv [hunk 3]
```diff
@@ -181,3 +181,86 @@ From;To;Type;Through;-1;-1;-1;-1;Comment
...
+502;499;major_river;1438;-1;-1;-1;-1;Waterspeak � Inkgill
+502;506;major_river;1438;-1;-1;-1;-1;Waterspeak � Muskpaw
+502;509;major_river;1438;-1;-1;-1;-1;Waterspeak � Hakshix
+502;478;major_river;1438;-1;-1;-1;-1;Waterspeak � Binan
+1190;1189;sea;1459;-1;-1;-1;-1;Vekam � Hukkirlam
+1190;1180;sea;1459;-1;-1;-1;-1;Vekam � Stinging Trail
+1190;1192;sea;1459;-1;-1;-1;-1;Vekam � Kypari Ik
+936;937;major_river;1424;-1;-1;-1;-1;Thundercleft � Krasarang
+934;865;major_river;1424;-1;-1;-1;-1;Narsong � Shen-zin
+604;557;major_river;1424;-1;-1;-1;-1;Fallingblossom � Cloudstrike
```
Comment (Kyraga) [inline map/adjacencies.csv:218]: 604;557;major_river;1440;-1;-1;-1;-1;Fallingblossom � Cloudstrike

**Pull request merged**